capirca/lib/aclgenerator.py

```diff
@@ -222,6 +222,7 @@ class ACLGenerator(object):
   # Only warn if these tokens are not implemented by a platform. These are not
   # meant to be overridden in subclasses like supported tokens/sub tokens.
   WARN_IF_UNSUPPORTED = {
+      'address_family',
```

Comment:
Could we rename this to either restrict_address_family or limit_address_family (other changes are also welcome)?

We have a lot of generators, where address family is used for a lot of fields (usually for the header's policy's address family), and this specific feature should be indicative that it only applies to terms and that it is restricting the address family. Any other variation that is semantically meaningful also works. Thanks!

Reply:
Good idea! Renamed to "restrict_address_family" as I couldn't find anything better. Thanks!